joplin/snippets/contact/models.py

```diff
@@ -36,78 +33,21 @@ class Contact(ClusterableModel):
     panels = [
         FieldPanel('name'),
         FieldPanel('email'),
-        InlinePanel('phone_number', label='Phone Numbers'),
-        PageChooserPanel('location_page'),
-        InlinePanel('hours', label='Hours'),
-        FieldPanel('hours_exceptions'),
+        InlinePanel('phone_numbers', label='Phone Numbers'),
         StreamFieldPanel('social_media'),
+        PageChooserPanel('location_page'),
     ]
 
     def __str__(self):
         return self.name
 
 
-class DayAndDuration(ClusterableModel):
```

Comment:
I started to wonder where this was moved to, then i realized contacts don't need to know this at all. 

v3 is going to be so much easier to follow